app/assets/stylesheets/base/_forms.scss

```diff
@@ -0,0 +1,95 @@
+$_form-box-shadow: inset 0 1px 3px rgba(#000, 0.06);
+$_form-box-shadow-focus: $_form-box-shadow, 0 0 5px rgba($action-color, 0.7);
```

Comment:
Why are the variables prefixed with an underscore?

Also, I would create another `library.scss` file to hold all your variables, instead of having them defined in multiple places across all the stylesheets.

Reply:
I would remove the underscore and move these variables to the library dir